### DISCUSSION TITLE
the 1 in text vs paragraph looks different. Is this intentional?

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#515] [General] [answered]
![image](https://user-images.githubusercontent.com/6718764/85077894-28fbbd80-b178-11ea-84b8-3e6b9bda6176.png)

**Comment by aayushpi** [comment]
@katieporter Can you investigate?. If I understand the question correctly, you are referring to the <Text> component in the first example? If so, what are you passing in the `as` prop for Text?

**Reply by elainelylee** [reply]
This is in Sketch. The first 1 is using "color-text" style, the second 1 is using "paragraph" style.

**Reply by corinne-nagel** [reply]
@katieporter circling back here, did you get a chance to check this out?

**Reply by kate-porter** [reply]
Taking a look now!

**Comment by kate-porter** [comment, marked as answer]
@elainelylee the `color-text` text style is configured to use monospaced numbers ([take a look here for more details on what that means](https://share.goabstract.com/7804bec5-7352-4336-827f-e6b4ff4a0730?mode=design&sha=36ad40021d8fe9c3b47ab9670da7d561b155adc5)).

To be totally honest, it's just a little bit tedious to adjust this for every symbol and text style, since the default in Sketch is not to use monospaced numbers. So, we only did it for `color-text`. That said, if this is blocking you on anything, we can definitely adjust it for `paragraph` as well. But if it's not blocking you, we will address this as we start building out the theme and symbols for UDL.

**Reply by elainelylee** [reply]
It's not blocking me. I'll make a note of it. Thanks
